superset-frontend/src/components/ListView/TableCollection.tsx

```diff
@@ -31,8 +31,120 @@ interface Props {
   loading: boolean;
 }
 
+const TableContainer = styled.div`
+  .table-cell-loader {
+    position: relative;
+
+    .loading-bar {
+      background-color: ${({ theme }) => theme.colors.secondary.light4};
+      border-radius: 7px;
+
+      span {
+        visibility: hidden;
+      }
+    }
+
+    &:after {
+      position: absolute;
+      transform: translateY(-50%);
+      top: 50%;
+      left: 0;
+      content: '';
+      display: block;
+      width: 100%;
+      height: 48px;
+      background-image: linear-gradient(
+        100deg,
+        rgba(255, 255, 255, 0),
+        rgba(255, 255, 255, 0.5) 60%,
+        rgba(255, 255, 255, 0) 80%
+      );
+      background-size: 200px 48px;
+      background-position: -100px 0;
+      background-repeat: no-repeat;
+      animation: loading-shimmer 1s infinite;
+    }
+  }
+
+  .actions {
+    white-space: nowrap;
+    font-size: 24px;
```

Comment:
Hmmmm... the theme has 
```  
      xl: 21,
      xxl: 28
```
... I wonder if we should use one of those, and possibly adjust the value if needed. The current theme values are based on the LESS variables file, and might need to be updated to more closely match SIP-34 specs.

Reply:
actually, I don't think this is even necessary anymore. This is left over from the days of the font-awesome icons.